src/rasterizer/display.py

```diff
@@ -61,6 +59,11 @@ def main(argv):
     else:
         characters_to_display = list(FLAGS.display)
 
+    # FIXME this for-try-except block be removed once Ostap finishes data
+    # serialization. We would just deserialize an entire font.
```

Comment:
@ostapstephan when data serialization is ready to go, let me know. I can remove a whole chunk of ugly code here once that is done.

Reply:
See #39